services/discord/discord.service.js

```diff
@@ -27,6 +32,9 @@ const documentation = `
   To use the Discord badge a Discord server admin must enable the widget setting on the server.
 </p>
 <iframe src="https://player.vimeo.com/video/364220040" width="640" height="210" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe>
+<p>
+  To get the total member count you need to create a invite that never ends, and then add the Invite ID after the Server ID in the URL
```

Comment:
Maybe we could be _extra_ clear by adding something along the lines of "If you do not include the Invite ID, only the number of members presently online will de displayed."

Reply:
Sure I'll add that in!